# 🔹 1. Arithmetic Operators

Used for math calculations.

In [1]:
x = 10
y = 3

print(x + y)   # 13 (addition)
print(x - y)   # 7  (subtraction)
print(x * y)   # 30 (multiplication)
print(x / y)   # 3.333... (division)
print(x // y)  # 3 (floor division → whole number)
print(x % y)   # 1 (modulus → remainder)
print(x ** y)  # 1000 (exponent → 10^3)

13
7
30
3.3333333333333335
3
1
1000


---

# 🔹 2. Comparison Operators

Used to compare values.

They return **True** or **False**.

In [2]:
a = 5
b = 10

print(a == b)   # False (equal to)
print(a != b)   # True  (not equal to)
print(a > b)    # False
print(a < b)    # True
print(a >= 5)   # True
print(b <= 9)   # False

False
True
False
True
True
False


---

# 🔹 3. Logical Operators

Combine multiple conditions.

In [3]:
x = 7

print(x > 5 and x < 10)   # True (both must be True)
print(x > 5 or x < 5)     # True (at least one True)
print(not(x > 5))         # False (negates the result)

True
True
False


---

# 🔹 Quick Summary Table

| Type | Operator | Meaning | Example (`x=10, y=3`) |
| --- | --- | --- | --- |
| Arithmetic | + | Addition | 10 + 3 → 13 |
|  | - | Subtraction | 10 - 3 → 7 |
|  | * | Multiplication | 10 * 3 → 30 |
|  | / | Division | 10 / 3 → 3.33 |
|  | // | Floor division | 10 // 3 → 3 |
|  | % | Modulus (remainder) | 10 % 3 → 1 |
|  | ** | Exponentiation | 10 ** 3 → 1000 |
| Comparison | == | Equal | 10 == 3 → False |
|  | != | Not equal | 10 != 3 → True |
|  | >, < | Greater, Less | 10 > 3 → True |
|  | >=, <= | Greater/less or equal | 10 >= 10 → True |
| Logical | and | Both True | True and False → False |
|  | or | One True | True or False → True |
|  | not | Negation | not True → False |


---

# 🔹 Practice 📝

Try these:

1. Check if a number is **even** (hint: use `%`).
2. Write a condition that checks if age is between 18 and 30.
3. Ask the user for two numbers → print which one is bigger.
4. Test if a number is **NOT** greater than 100.

---